src/data_morph/data/loader.py

```diff
@@ -53,6 +53,7 @@ class DataLoader:
         'SDS': 'superdatascience.csv',
         'sheep': 'sheep.csv',
         'soccer': 'soccer_ball.csv',
+        'gorilla': 'gorilla.csv',
```

Comment:
Let's move this after `dog` and before `music` so it is in alphabetical order.

Reply:
OK, fixed it!